content/docs/dvclive/api-reference/live/log_sklearn_plot.md

```diff
@@ -63,19 +49,19 @@ plot.
 
 Calls
 [sklearn.calibration.calibration_curve](https://scikit-learn.org/stable/modules/generated/sklearn.calibration.calibration_curve.html)
-and stores the data at `{Live.dir}/plots/calibratrion.json` in a format
+and stores the data at `{Live.dir}/plots/sklearn/calibratrion.json` in a format
 compatible with `dvc plots`.
 
 ```py
 y_true = [0, 0, 1, 1]
 y_score = [0.1, 0.4, 0.35, 0.8]
-live.log_plot("calibration", y_true, y_score)
+live.log_sklearn_plot("calibration", y_true, y_score)
 ```
 
 Example usage with `dvc plots`:
 
-```dvc
-$ dvc plots show 'dvclive/plots/calibration.json' \
+```cli
+$ dvc plots show 'dvclive/plots/sklearn/calibration.json' \
 -x prob_pred -y prob_true \
 --x-label 'Mean Predicted Probability' \
 --y-label 'Fraction of Positives' \
```

Comment:
⚠️ **This is a blocker**: All the plots are missing in heroku except for precision recall